draft-js-anchor-plugin/src/components/Link/index.js

```diff
@@ -1,4 +1,4 @@
-import React, { PropTypes } from 'react';
```

Comment:
React needs to be declared in scope, because jsx...

Reply:
Whoops - missed that. Recommitted + squashed

Reply:
@stevewillard sweet thx, will merge after tests pass